src/client/src/system/SystemButtonRenderer.cpp

```diff
@@ -200,8 +200,8 @@ void SystemButtonRenderer::updateButtons(irr::video::IVideoDriver *driver)
 				}
 			}
 			else {
-				for (int j = 0; j < buttons.size(); j++) {
-					buttons[j].triggered == true;
+				for (uint32_t j = 0; j < buttons.size(); j++) {
+					buttons[j].triggered |= true;
```

Comment:
This one is an actual change, surely before it wasn't working?

Reply:
This was sure to a build build warning occurring when they had just an `==`.. I can't remember the exact warning but this fixed it. 
In any case, this should have the same effect on a boolean which `buttons[j].triggered` is AFAIR.

Reply:
Wait what the hell am I talking about.. One sec

Reply:
Yeah I think it must have been broken before - looking into this properly now.

Reply:
Ok, upon further examination and a chat with @grant-m-s, it's mean to just be a single equals sign. This is equivalent to `|=`, so to save another commit I'm going to call this good to go.

(Unless you massively object to using `|=`)